sdcm/nemesis.py

```diff
@@ -632,22 +632,21 @@ def disrupt_nodetool_refresh(self, big_sstable=True, skip_download=False):
         result = self.target_node.run_nodetool(sub_cmd="cfstats", args="keyspace1.standard1")
 
         def do_refresh(node):
-            if not skip_download:
-                key_store = KeyStore()
-                creds = key_store.get_scylladb_upload_credentials()
-                # Download the sstable files from S3
-                remote_get_file(node.remoter, sstable_url, sstable_file,
-                                hash_expected=sstable_md5, retries=2,
-                                user_agent=creds['user_agent'])
-            result = node.remoter.run("sudo ls -t /var/lib/scylla/data/keyspace1/")
+            key_store = KeyStore()
+            creds = key_store.get_scylladb_upload_credentials()
+            # Download the sstable files from S3
+            remote_get_file(node.remoter, sstable_url, sstable_file,
+                            hash_expected=sstable_md5, retries=2,
+                            user_agent=creds['user_agent'])
+            result = node.remoter.sudo("ls -t /var/lib/scylla/data/keyspace1/")
             upload_dir = result.stdout.split()[0]
-            node.remoter.run('sudo -u scylla tar xvfz {} -C /var/lib/scylla/data/keyspace1/{}/upload/'.format(
-                sstable_file, upload_dir))
+            node.remoter.sudo(
+                f'tar xvfz {sstable_file} -C /var/lib/scylla/data/keyspace1/{upload_dir}/upload/', user='scylla')
```

Comment:
I think it's better to add `if node.is_docker():` condition here as for housekeeping mark and don't override `remoter.sudo()`  method it k8s remoter.

BTW, it's only two places in our codebase where we run `sudo -u`. Both times `sudo -u scylla`. Can you agree that it doen't worth of "remoter.sudo()" method complications?